app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
```

Comment:
Haha, even further reason to improve the copy around it! Might be good for you to check the form too to confirm that's clear enough, it's grouped under the Artist section so hopefully it's covered. 

Artist's Background = 👍 !